.eslintrc.js

```diff
@@ -1,13 +1,21 @@
 module.exports = {
   ignorePatterns: ['**/*.css.ts'],
-  extends: ['eslint:recommended', 'plugin:prettier/recommended'],
+  extends: [
+    'eslint:recommended',
+    'plugin:react/recommended',
+    'plugin:prettier/recommended',
+    'plugin:storybook/recommended',
+  ],
   parser: '@babel/eslint-parser',
   parserOptions: {
     ecmaVersion: 2020,
     sourceType: 'module',
     ecmaFeatures: {
       jsx: true,
     },
+    babelOptions: {
+      presets: ['@babel/preset-react'],
+    },
```

Comment:
This was a necessary addition to get JSX playing nicely; the `ecmaFeatures: { jsx: true } }` wasn't sufficient on its own.